pytorch_lightning/utilities/hydra.py

```diff
@@ -0,0 +1,71 @@
+import os
+from typing import List, Optional, Tuple
+
+import __main__
+import torch
+
+from pytorch_lightning.utilities import _HYDRA_AVAILABLE
+
+if _HYDRA_AVAILABLE:
+    from hydra.core.hydra_config import HydraConfig
+    from hydra.utils import get_original_cwd
+    from omegaconf import OmegaConf
+
+
+def get_ddp_spawn_command_for_hydra(command: List[str], local_rank: str) -> Tuple[str, str]:
+    """Modifies the DDP spawn command to support Hydra initiated processes.
+
+    If Hydra is initialized:   1) Set `cwd` to the hydra working directory   2) Use the stored configuration in
+    `hydra_cfg.output_subdir / config.yaml` to spawn a new child
+    """
+
+    cwd: Optional[str] = None
+    if HydraConfig.initialized():
+        orig_cwd = get_original_cwd()
+        cwd = os.getcwd()
+        os_cwd = f'"{cwd}"'  # this is needed to handle characters like `=` in the directory name
+
+        hydra_cfg = HydraConfig.get()
+        hydra_output = os.path.join(os.path.relpath(cwd, orig_cwd), hydra_cfg.output_subdir)
```

Comment:
Why is `relpath` necessary?

Reply:
I'm looking into that, I don't think it will be needed.